theme6/source/partials/_offers.erb

```diff
@@ -0,0 +1,34 @@
+<section class="featured-kits">
```

Comment:
Nice partial 👍 Very simple and clean. Keep doing this. As partials should not be too extensive.